tests/pds/pds_storagesize_increase_test.go

```diff
@@ -353,21 +376,24 @@ var _ = Describe("{ScaleUpDsPostStorageSizeIncreaseVariousRepl}", func() {
 						})
 						stepLog = "Verify storage size before and after storage resize - Verify at STS, PV,PVC level"
 						Step(stepLog, func() {
-							err := ValidateDepConfigPostStorageIncrease(ds, updatedDeployment, stConfigModelUpdated1, resConfigModelUpdated1, initialCapacity, updatedPvcSize)
+							err := ValidateDepConfigPostStorageIncrease(ds, updatedDeployment, stConfigModelUpdated1, resConfigModelUpdated1, initialCapacity, updatedPvcSize, beforeResizePodAge)
 							log.FailOnError(err, "Failed to validate DS Volume configuration Post Storage resize")
 						})
 					})
 					stepLog = "Increase the storage size again after Scale-UP"
 					Step(stepLog, func() {
 						newTemplateName2 := "autoTemp-" + strconv.Itoa(rand.Int())
+						currStorageSize := *resConfigModelUpdated1.StorageRequest
+						newStorageSize, _ := strconv.Atoi(currStorageSize[:len(currStorageSize)-1])
+						updatedStorage := strconv.Itoa(newStorageSize+10) + "G"
 						updatedTemplateConfig2 := controlplane.Templates{
```

Comment:
esConfigModelUpdated1
newTemplateName2
instead of 1 and 2, can we have names like before and after? for more readability
also, updatedStorage is inceasedStorage?